src/material-experimental/mdc-button/button.scss

```diff
@@ -106,3 +106,26 @@
   right: $offset;
   border-width: $offset;
 }
+
+// By default, all focus indicators are flush with the bounding box of their
+// host element. For the button element, default inset/offset
```

Comment:
I don't think we need to repeat the "By default, all focus indicators are flush with the bounding box of their host element." blurb across all components, I'd vote to remove it.